packages/addon-modernjs/src/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface AddonOptions {
+  /**
+   *  `path` of Modern.js config file
+   * @default undefined
```

Comment:
```suggestion
   * @default "modern.config.ts"
```

`undefined` will make user confused, and actually we're using `modern.config.ts` as default.

Reply:
Done~